project_buttom_kanban_open_form_view/models/project_project.py

```diff
@@ -0,0 +1,16 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, api, fields
```

Comment:
Not necessary api and fields imports
```suggestion
from odoo import models
```